### ISSUE TITLE
pretrain problem

### ISSUE THREAD

**Issue by aopolin-lv** [FDChongLi/TwoWaysToImproveCSC#3]
您好：

       请问在预训练阶段的时候：
       1. 你们使用了多少规模的语料(如一千万条句子这种)
       2. learning rate的设置是多少
       3. 跑了多少steps
       4. pretrain的子任务是和bert保持一致，还是和csc的任务保持一致

       十分感谢！

**Comment by FDChongLi** [comment]
1. 预训练语料收集于Weibo和Wikipedia，包含大约930万条句子
2. BERT和Soft-masked BERT的学习率都为2e-5，而SpellGCN的学习率是5e-5
3. 预训练中的epoch数量取决于模型在验证集上的效果
4. 和csc的任务保持一致
